src/track/keyutils.cpp

```diff
@@ -247,8 +247,17 @@ QString KeyUtils::getGlobalKeyText(const Keys& keys, KeyNotation notation) {
 }
 
 // static
-ChromaticKey KeyUtils::guessKeyFromText(const QString& text) {
-    QString trimmed = text.trimmed();
+ChromaticKey KeyUtils::guessKeyFromText(QString text) {
+    // Remove Shift (Tuning) Information used by Rapid Evolution like: "A#m +50";
+    int shiftStart = text.indexOf('+');
+    if (shiftStart < 0) {
+        shiftStart = text.indexOf('-');
+    }
+    if (shiftStart >= 0) {
+        text = text.left(shiftStart);
+    }
+    const QString trimmed = text.trimmed();
```

Comment:
I'd personally favor to change the regex's instead, but this is fine as stopgap solution if you insist.

Reply:
Yes, once we are aware of tuning this needs to be replaced. 

While the tuning issue exists, luckily most tracks are at A = 440 Hz. So this is something that is only reasonable with detector support like in Rapid Evolution.   

A prominent out of tune Track is "Coldplay - The Scientist" which is recorded with an out of tune piano at A = 446 Hz
https://www.reddit.com/r/Coldplay/comments/8tjhwy/nearly_every_song_on_arobth_is_out_of_tune_or_not/